Cargo.toml

```diff
@@ -35,20 +35,20 @@ library = []
 
 [dependencies]
 bech32 = "=0.8.1"
-provwasm-std = { version = "=1.1.2" }
-cosmwasm-std = { version = "=1.2.0" }
-cosmwasm-storage = { version = "=1.2.0" }
-cw-storage-plus = "=0.12.1"
+provwasm-std = { version = "=2.4.0" }
+cosmwasm-schema = { version = "=2.1.4" }
+cosmwasm-std = { version = "=2.1.4" }
+cosmwasm-storage = { version = "=1.5.2" }
+cw-storage-plus = "=2.0.0"
 os-gateway-contract-attributes = "=1.0.1"
 result-extensions = "=1.0.2"
-schemars = "=0.8.11"
+schemars = "=0.8.21"
 semver = "=1.0.7"
-serde = { version = "=1.0.137", default-features = false, features = ["derive"] }
-serde-json-wasm = { version = "=0.4.1" }
-thiserror = { version = "=1.0.26" }
+serde = { version = "=1.0.213", default-features = false, features = ["derive"] }
```

Comment:
Were things breaking before this downgrade?

Reply:
Unsure which dependency you mean by downgrade, as I think these were all upgrades, but I believe the updates I made here to the packages other than `provwasm` and `cosmwasm` were all required to satisfy `provwasm` version constraints.

Reply:
That's my bad.  I thought I saw 1.0.113 or something 🤦   Resolved